### PR TITLE
BAM-1599 Linkmate toggle documentation

### PR DESCRIPTION
**Pull request by adachen**

### CODE REVIEW

File: docs/linkmate.rst [hunk 1]
```diff
@@ -13,6 +13,23 @@ Narrativ platform.
 Your account must be a member of our LinkMate program to use this feature.
 For more information about this program, feel free to `contact us`_.
 
+Toggle LinkMate
```
Comment (csdev) [inline docs/linkmate.rst]: This belongs with the tag instructions, not the API endpoint documentation.

File: docs/tagpublisher.rst [hunk 2]
```diff
@@ -36,3 +36,23 @@ in the HEAD section of all your site's pages.
...
 
 .. _setup: https://dashboard.narrativ.com/#/publisher/account/setup
+
+Usage
+-----
+
+Toggle LinkMate
+^^^^^^^^^^^^^^^
+
+* Enable or disable LinkMate on a page by updating the Narrativ window object with true or false::
```
Comment (csdev) [inline docs/tagpublisher.rst]: Add some context here so non-LinkMate publishers don't get confused when reading this. For example, see https://github.com/hellonarrativ/api-docs/blob/193d59f497cf051dedd296e4df1e2f2621aa0882/docs/tagpublishercustom.rst#linkmate-automatic-smartlink-retrieval and https://github.com/hellonarrativ/api-docs/blob/193d59f497cf051dedd296e4df1e2f2621aa0882/docs/linkmate.rst#usage

File: docs/tagpublisher.rst [hunk 3]
```diff
@@ -36,3 +36,23 @@ in the HEAD section of all your site's pages.
...
+
+    window.NRTV_EVENT_DATA.donotlink = true;
+
+* Disable LinkMate on a specific link by adding #donotlink to the URLs::
+
+    http://merchant.example/product/name#donotlink
+
+* Single match a link by adding #locklink to the URLs. In this case, the link can only be matched
```
Comment (csdev) [inline docs/tagpublisher.rst]: Don't use internal terminology here. "Merchant Product" and "canonical host" are confusing to ordinary readers. Try to be consistent with the documentation for `exclusive_match_requested` here: https://github.com/hellonarrativ/api-docs/blob/193d59f497cf051dedd296e4df1e2f2621aa0882/docs/smartlink.rst#request

File: docs/tagpublisher.rst [hunk 4]
```diff
@@ -36,3 +36,23 @@ in the HEAD section of all your site's pages.
 
 
 .. _setup: https://dashboard.narrativ.com/#/publisher/account/setup
+
+Usage
+-----
+
+Toggle LinkMate
+^^^^^^^^^^^^^^^
+
```
Comment (myfashionhub) [inline docs/tagpublisher.rst]: The first way to turn on Linkmate for a publisher can be done by a Narrativ admin (via the Manage account modal)

File: docs/tagpublisher.rst [hunk 5]
```diff
@@ -36,3 +36,30 @@ in the HEAD section of all your site's pages.
...
+:doc:`LinkMate API <linkmate>` for your account or specific pages to retrieve the SmartLinks for a set of URLs, 
+automatically creating new SmartLinks that do not already exist. You can also disable LinkMate for your account,
+specific pages or links to prevent creating SmartLinks for certain URLs.
+
+Toggle LinkMate
+^^^^^^^^^^^^^^^
+
+* Enable or disable LinkMate for your publisher account through a Narrativ Admin
+
+* Enable or disable LinkMate on a page by updating the Narrativ window object with true or false::
```
Comment (myfashionhub) [inline docs/tagpublisher.rst]: remove `with true or false`

**Pull request merged**